src/App.js

```diff
@@ -11,33 +11,128 @@ class App extends React.Component {
+      currCards: [],
+      hasGameStarted: false,
+      roundWinner: null,
+      player1Total: 0,
+      player2Total: 0,
+    });
+  };
+
   render() {
-    const currCardElems = this.state.currCards.map(({ name, suit }) => (
-      // Give each list element a unique key
-      <div key={`${name}${suit}`}>
-        {name} of {suit}
-      </div>
-    ));
+    // display cards
+    const currCardElements = this.state.currCards.map(
+      ({ name, suit }, index) => (
+        // Give each list element a unique key
+        <div key={`${name}${suit}`}>
+          {/* retrieve card images */}
+          <img
+            src={require(`../src/assets/${name}Of${suit}.png`)}
+            alt="cardImage"
+          />
+          <p>
+            Player {index + 1}'s card is {name} of {suit}
+          </p>
+        </div>
+      ),
+    );
+    // round winner's messages (2 expectations)
+    const roundWinnerMessage = this.state.roundWinner
+      ? `Player ${this.state.roundWinner} won this round.`
+      : `No winner this round. The game continues!`;
+
+    // individual winner's message
+    // if player1CurrentScore > player2CurrentScore, player 1 wins, else player 2 wins
+    const winnerMessage =
+      this.state.player1CurrentScore > this.state.player2CurrentScore
+        ? `Player 1 wins the game`
+        : `Player 2 wins the game`;
+
+    // players' current and total scores
+    const player1CurrentRoundScore = `Player 1's current round score is: ${this.state.player1CurrentScore}`;
+    const player2CurrentRoundScore = `Player 2's current round score is: ${this.state.player2CurrentScore}`;
+    const player1TotalScore = `Player 1 won a total of: ${this.state.player1Total} rounds.`;
+    const player2TotalScore = `Player 2 won a total of: ${this.state.player2Total} rounds.`;
+
+    // Deal button text changes at end of game to start again
+    const numRoundsLeft = this.state.cardDeck.length / 2;
+    const dealButtonText = numRoundsLeft === 0 ? "Reset Game" : "Deal";
 
     return (
       <div className="App">
         <header className="App-header">
           <h3>High Card 🚀</h3>
-          {currCardElems}
+          {currCardElements}
+          <br />
+          {/* Button changes functionality depending on game state */}
+          <button
+            onClick={numRoundsLeft === 0 ? this.resetGame : this.dealCards}
+          >
+            {dealButtonText}
+          </button>
           <br />
-          <button onClick={this.dealCards}>Deal</button>
+          {/* to display messages */}
+          <p>{this.state.gameStart && roundWinnerMessage}</p>
+          <p>{this.state.gameStart && player1CurrentRoundScore}</p>
+          <p>{this.state.gameStart && player1TotalScore}</p>
+          <p>{this.state.gameStart && player2CurrentRoundScore}</p>
+          <p>{this.state.gameStart && player2TotalScore}</p>
+
+          {/* Render winner message if the game is over */}
+          <p>{numRoundsLeft === 0 && winnerMessage}</p>
```

Comment:
I would either go 
```suggestion
          <p>{!numRoundsLeft && winnerMessage}</p>
```
or 
```suggestion
          <p>{!this.state.cardDeck.length && winnerMessage}</p>
```